app/components/UserContributionsList.js

```diff
@@ -1,13 +1,38 @@
 import React from 'react'
 import styled from 'styled-components/native'
+import Container from '../components/Container'
 
 import UserContributionsItem from './UserContributionsItem'
 
+const TextContainer = styled.View`
+  flex: 1;
+  align-items: center;
+  justify-content: center;
+`
+
+const Text = styled.Text`
```

Comment:
You can remove this `Text` styling - the styles are set globally in the style PR I pushed up